### PR TITLE
nuxtjs example

### PR DESCRIPTION
**Pull request by vitalikda**
# Description

Example of Nuxt 3 and Grafbase. 
The GraphQL endpoint is guarded by serverless function to prevent exposure of the private key. 

Outdated! ~~Live Demo: https://grafbase-nuxtjs-example.vercel.app/~~

Notes: 
- nuxt community favors unocss instead of tailwind, therefore I went with it. Unocss supports tailwind classes, so the only difference is less setup boilerplait.
- `nuxtjs/auth` is incompatible with Nuxt 3 yet, therefore custom implementation of GitHub auth: `src/composables/auth.ts`, `src/middleware/auth.global.ts` and `src/server/api/github/callback.ts`

# Type of change

- [ ] 💔 Breaking
- [ ] 🚀 Feature
- [ ] 🐛 Fix
- [ ] 🛠️ Tooling
- [ ] 🔨 Refactoring
- [ ] 🧪 Test
- [ ] 📦 Dependency
- [x] 📖 Requires documentation update

### CODE REVIEW

File: examples/nuxtjs/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Grafbase ⨯ Nuxt 3
+
+[Join our Discord](https://discord.gg/grafbase)
```
Comment (fbjork) [inline examples/nuxtjs/README.md:3]: Perhaps change to Join our Community and link to grafbase.com/community?

File: examples/nuxtjs/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+[![Deploy to Grafbase](https://grafbase.com/button)](https://grafbase.com/new/configure?template=Todo&source=https%3A%2F%2Fgithub.com%2Fgrafbase%2Fgrafbase%2Ftree%2Fmain%2Ftemplates%2Ftodo)
+</br>
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/import/project?template=https://github.com/grafbase/grafbase/tree/main/examples/nextjs)
```
Comment (fbjork) [inline examples/nuxtjs/README.md:7]: Deploy to Vercel?

File: examples/nuxtjs/grafbase/schema.graphql [hunk 3]
```diff
@@ -0,0 +1,12 @@
+type TodoList @model {
```
Comment (fbjork) [inline examples/nuxtjs/grafbase/schema.graphql:1]:
```type TodoList @model {
  id: ID!
  title: String!
  todos: [Todo]
}

type Todo @model {
  id: ID!
  list: TodoList
  title: String!
  complete: Boolean!
}```

**Comment by jvidalv** [comment]
Remember to rename everything (folder included) to Nuxt instead of NuxtJs

File: examples/nuxt/README.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# Grafbase ⨯ Nuxt 3
+
+[Join our Community](https://grafbase.com/community)
+
+[![Deploy to Grafbase](https://grafbase.com/button)](https://grafbase.com/new/configure?template=Todo&source=https%3A%2F%2Fgithub.com%2Fgrafbase%2Fgrafbase%2Ftree%2Fmain%2Ftemplates%2Ftodo)
+</br>
+[![Deploy to Vercel](https://vercel.com/button)](https://vercel.com/import/project?template=https://github.com/grafbase/grafbase/tree/main/examples/nextjs)
```
Comment (notrab) [inline examples/nuxt/README.md:7]:
```suggestion
[![Deploy to Vercel](https://vercel.com/button)](https://vercel.com/import/project?template=https://github.com/grafbase/grafbase/tree/main/examples/nuxt)
```

File: examples/nuxt/README.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+- [Grafbase](https://grafbase.com/) - learn about Grafbase features and API.
+
+To learn more about Nuxt 3, take a look at the following resources:
+
+- [Nuxt 3 Documentation](https://v3.nuxtjs.org/guide/concepts/introduction) - learn about Nuxt 3 features and API.
+
+### Run on CodeSandbox
+
+[![Develop with CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://githubbox.com/grafbase/grafbase/tree/main/examples/nuxtjs)
```
Comment (notrab) [inline examples/nuxt/README.md:31]:
```suggestion
[![Develop with CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://githubbox.com/grafbase/grafbase/tree/main/examples/nuxt)
```

Comment (notrab) [inline examples/nuxt/README.md:31]:
```suggestion
[![Develop with CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/grafbase/grafbase/tree/main/examples/nuxt)
```

Comment (notrab) [inline examples/nuxt/README.md:31]:
```suggestion
[![Develop with CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://githubbox.com/grafbase/grafbase/tree/main/examples/nuxt)
```

**Review by fbjork — APPROVED** [review]

**Pull request merged**